GiniVision/Classes/Core/GiniScreenAPICoordinator.swift

```diff
@@ -235,45 +235,53 @@ extension GiniScreenAPICoordinator: UINavigationControllerDelegate {
 // MARK: - Camera Screen
 
 extension GiniScreenAPICoordinator: CameraViewControllerDelegate {
-    func camera(_ viewController: CameraViewController, didCaptureDocuments documents: [GiniVisionDocument]) {
+    func camera(_ viewController: CameraViewController,
```

Comment:
This method is so long and covers so many cases that it gives me a headache.

Reply:
Yes it is. I thought about splitting it but it can be more confusing for those who implement the Component API.

Reply:
They would have to implement several methods that would lead to the same target, process a document.

Or you mean just our implementation? Then it would be possible to split into different methods.

Reply:
yeah, just the implementation.